lib/author.rb

```diff
@@ -0,0 +1,17 @@
+class Author
+
+  attr_reader :first_name, :last_name, :name, :books
```

Comment:
Since we only need to call `.name` from outside the class, we don't need `.first_name` or `.last_name` as attr_readers